uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                            .translate(Point::new(76, 2)),
+                    ),
+                    GamePeriod::OvertimeSecondHalf => display.draw(
+                        Font8x15::render_str("O/T 2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::PreSuddenDeath => display.draw(
+                        Font8x15::render_str("PRE-SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                    GamePeriod::SuddenDeath => display.draw(
+                        Font8x15::render_str("SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_1")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                }
+            }
+            _ => {
+                display.draw(
+                    Font16x31::render_str(&secs_to_time_string(state.secs_in_period))
+                        .stroke(Some(game_color))
+                        .translate(Point::new(108, 33)),
+                );
+                match state.current_period {
+                    GamePeriod::FirstHalf => {
+                        display.draw(
+                            Font8x15::render_str("1ST")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(72, 33)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("HALF")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(68, 48)),
+                        );
+                    }
+                    GamePeriod::SecondHalf => {
+                        display.draw(
+                            Font8x15::render_str("2ND")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(72, 33)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("HALF")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(68, 48)),
+                        );
+                    }
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_2")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                }
+                match state.timeout {
+                    TimeoutState::White(_) => {
```

Comment:
Switch to `TimeoutState::White(secs) => {` ...

Reply:
is this true for all instances of `(_)`?

Reply:
corresponds with the change in the next comment

Reply:
`_` means "I don't care what this value is" `secs` means "I want to store this value to use later, and call it `secs`"